config/creatrs/archer.cfg

```diff
@@ -88,8 +88,8 @@ ShotOrigin = 0 0 384
 
 [experience]
 ; Creature powers (spells), and the creature level at which they're given (max 10 spells).
-Powers = SWING_WEAPON_FIST FIRE_ARROW NULL NAVIGATING_MISSILE NULL SPEED NULL SLOW NULL NULL
-PowersLevelRequired = 1 1 0 4 0 6 0 8 0 0
+Powers = FIRE_ARROW NULL NULL NAVIGATING_MISSILE NULL SPEED NULL SLOW NULL NULL
```

Comment:
Because of the recent change that it is now possible to make spells strength-based too. Now the fire_arrow is no longer a fixed-damage spell, but an ability that grows along with strength, and as such properly replaces the fist.

Reply:
ok